src/client/pages/Geo/utils/forestEstimations.ts

```diff
@@ -43,12 +49,11 @@ export const builForestEstimationsDataTable = async (
         area: Number(area.toFixed(2)),
         fra1ALandAreaPercentage: Number(percentage.toFixed(2)),
         sourceKey: key,
-        sourceName,
+        sourceLabelKey,
       })
     } else {
       hansenPercentages.forEach((number: number) => {
-        const sourceNameKey = metadata.titleKey ?? key
-        const sourceName = `${i18n.t(sourceNameKey)} ${number} %`
```

Comment:
Instead of storing the translated text in Redux, we are now storing the translation keys. The `hansenPercent` property is added so it can be included when the translation is performed, instead of hardcoding it like in this line.